javascript-source/core/commandCoolDown.js

```diff
@@ -31,19 +31,41 @@
 
     $.raffleCommand = null;
 
+    const   Type = {
+                User: 'user',
+                Global: 'global',
```

Comment:
Codacy has a fix for the issue: [Replace `················Global:·'global'` with `······Global:·"global"`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
      Global: "global",
```